app/helpers/application_helper.rb

```diff
@@ -2,10 +2,22 @@
 
 module ApplicationHelper
   def profile_dashboard_url(user)
-    if user.admin_profile
+    if user.admin?
       admin_suppliers_url
+    elsif user.induction_coordinator?
+      induction_coordinator_dashboard_url(user)
     else
       dashboard_url
     end
   end
+
+private
+
+  def induction_coordinator_dashboard_url(user)
+    school = user.induction_coordinator_profile.schools.first
```

Comment:
Do we have a ticket about one induction coordinator having multiple schools? Is that even on our radar?

Reply:
Alaric said it's unlikely but that the first thing would be a school picker screen if that were to happen. We're deliberately omitting it for now